src/MoreVehicles/MoreVehiclesMod.cs

```diff
@@ -113,6 +116,18 @@ public override void OnLevelLoaded(LoadMode mode)
                     break;
 
                 default:
+                    if (!isWorkshopMode || patcher == null)
```

Comment:
We actually don't need these additional checks here.
If `isWorkshopMode` is `false`, then `patcher` will be `null`, and this is checked at the start of the method.